script/tool/lib/src/create_all_plugins_app_command.dart

```diff
@@ -79,7 +79,13 @@ class CreateAllPluginsAppCommand extends PluginCommand {
 
     final StringBuffer newGradle = StringBuffer();
     for (final String line in gradleFile.readAsLinesSync()) {
-      newGradle.writeln(line);
+      if (line.contains('minSdkVersion 16')) {
```

Comment:
Since we are creating "all plugins", which def contains google maps and web views, should we just never allow minSdkVersion to 16?

Reply:
In the test? Yup, this line changes the `minSdkVersion`.   For all Flutter, see https://github.com/flutter/flutter/issues/82000